src/ui/Recommendation/Recommendation.ts

```diff
@@ -127,8 +127,10 @@ export class Recommendation extends SearchInterface implements IComponentBinding
      * However, setting this option to `false` can be useful to display side results in a search page.
      *
      * Default value is `true`.
+     *
+     * @deprecated This option is now deprecated. The correct way to control this behaviour is to configure an appropriate machine learning model in the administration interface (Recommendation, Relevance tuning, Query suggestions)
```

Comment:
* behaviour -> behavior (both forms are correct, but we normally use the latter one (American English).
* Add a period (.) at the end of the sentence: 

> (...), Query suggestions).

Otherwise, it's all good!